plugins/emc/emc_vnx_file_

```diff
@@ -0,0 +1,627 @@
+		echo "${server}_${elements_array[i*$NUMCOL]}".value "${elements_array[i*$NUMCOL+4]}"
+	done
+
+	elements_array=()
+
+#nfs.client data
+# [nasadmin@mnemonic0 ~]$ server_stats server_2 -monitor nfs.client -count 1 -terminationsummary no 
+# server_2                    Client                     NFS      NFS      NFS       NFS        NFS      NFS      NFS       NFS   
+# Timestamp                                             Total    Read     Write   Suspicious   Total    Read     Write      Avg   
+#                                                       Ops/s    Ops/s    Ops/s    Ops diff    KiB/s    KiB/s    KiB/s   uSec/call
+# 20:26:38   id=192.168.1.223                           2550       20     2196          13     4673      159     4514       1964
+#            id=192.168.1.2                              691        4        5           1     1113      425      688       2404
+#            id=192.168.1.1                              159        0        0          51        0        0        0       6017
+#            id=192.168.1.6                               37        4        2           0      586      295      291       5980
+#            id=192.168.1.235                             21        1        0           0        0        0        0     155839
+#            id=192.168.1.224                              5        0        5           0       20        0       20     704620
+
+	echo -e "\nmultigraph vnx_emc_nfs_client_ops_s"
+ 	member_elements_by_line=$(run_remote server_stats server_2 -monitor nfs.client -count 1 -terminationsummary no -titles never | sed -ne 's/^.*id=//p')
+	NUMCOL=9
+	LINES=$(wc -l <<< "$member_elements_by_line")
+	while IFS=$'\n' read -ra graphs; do
+		elements_array+=($graphs)
+	done  <<< "$member_elements_by_line"
+
+	#Not drawing elements in case of empty set
+	if [ "${#elements_array[@]}" -eq "0" ]; then  LINES=0; fi
+
+	for (( i=0; i<$((LINES)); i++ )); do
+		client="$(clean_fieldname "_${elements_array[i*$NUMCOL]}")"
+		echo "${server}${client}_r".value "${elements_array[$i*$NUMCOL+2]}"
+		echo "${server}${client}_w".value "${elements_array[$i*$NUMCOL+3]}"
+		echo "${server}${client}_t".value "${elements_array[$i*$NUMCOL+1]}"
+		echo "${server}${client}_s".value "${elements_array[$i*$NUMCOL+4]}"
+        done
+	echo -e "\nmultigraph vnx_emc_nfs_client_b_s"
+	for (( i=0; i<$((LINES)); i++ )); do
+		client="$(clean_fieldname "_${elements_array[i*$NUMCOL]}")"
+		echo "${server}${client}_r".value "$((${elements_array[$i*$NUMCOL+6]} * 1024))"
+		echo "${server}${client}_w".value "$((${elements_array[$i*$NUMCOL+7]} * 1024))"
+		echo "${server}${client}_t".value "$((${elements_array[$i*$NUMCOL+5]} * 1024))"
+        done
+	echo -e "\nmultigraph vnx_emc_nfs_client_avg_usec"
+	for (( i=0; i<$((LINES)); i++ )); do
+		client="$(clean_fieldname "_${elements_array[i*$NUMCOL]}")"
+		echo "${server}${client}".value "${elements_array[$i*$NUMCOL+8]}"
+	done
+	fi
+done
+if [ -z $nas_server_ok ]; then
```

Comment:
Please assign an initial value to the variable in order to avoid shellcheck warnings (and potentially allow `error on unset` mode).